src/entries/popup/components/ExternalImage/ExternalImage.tsx

```diff
@@ -60,7 +61,13 @@ const ExternalImage = (
 
   if (!signedUrl) return null;
   return (
-    <Box style={{ overflow: 'clip' }} borderRadius={props.borderRadius}>
+    <Box
+      style={{
+        overflow: 'clip',
+        maxHeight: props.maxHeight ? props.maxHeight : '',
```

Comment:
I think we can safely set `maxHeight` to `height`, so we don't need the extra `maxHeight` prop

Reply:
ok cool. wasn't sure if this was a good idea or not / had other implications elsewhere that i hadn't considered.

Reply:
fixed.